src/standalone/serviceRegistry.node.ts

```diff
@@ -85,4 +86,13 @@ export function registerTypes(context: IExtensionContext, serviceManager: IServi
 
     // Dev Tools
     registerDevToolTypes(context, serviceManager, isDevMode);
+
+    const configuration = serviceManager.get<IConfigurationService>(IConfigurationService);
+    if (configuration.getSettings().kernelPickerType === 'Insiders') {
```

Comment:
We are checking this in multiple places, wish we can have a more elegant approach ;(

Reply:
I wanted to create a single class with a single readonly property,